packages/nx/src/command-line/import/utils/prepare-source-repo.ts

```diff
@@ -19,56 +19,25 @@ export async function prepareSourceRepo(
     join(gitClient.root, source)
   );
 
-  if (relativeSourceDir !== '') {
-    if (await gitClient.hasFilterRepoInstalled()) {
-      spinner.start(
-        `Filtering git history to only include files in ${relativeSourceDir}`
-      );
-      await gitClient.filterRepo(relativeSourceDir);
-    } else {
-      spinner.start(
-        `Filtering git history to only include files in ${relativeSourceDir} (this might take a few minutes -- install git-filter-repo for faster performance)`
-      );
-      await gitClient.filterBranch(relativeSourceDir, tempImportBranch);
-    }
-    spinner.succeed(
-      `Filtered git history to only include files in ${relativeSourceDir}`
+  if (await gitClient.hasFilterRepoInstalled()) {
+    spinner.start(
+      `Filtering git history to only include files in ${relativeSourceDir}`
     );
-  }
-
-  const destinationInSource = join(gitClient.root, relativeDestination);
-  spinner.start(`Moving files and git history to ${destinationInSource}`);
-
-  // The result of filter-branch will contain only the files in the subdirectory at its root.
-  const files = await gitClient.getGitFiles('.');
```

Comment:
NOTE: Moving files is no longer needed since both `git filter-repo` and `git filter-branch` can handle the move automatically, in a way that allows for rebase.